frigate/track/norfair_tracker.py

```diff
@@ -55,15 +55,20 @@ def frigate_distance(detection: Detection, tracked_object) -> float:
 
 
 class NorfairTracker(ObjectTracker):
-    def __init__(self, config: CameraConfig, ptz_autotracker_enabled, ptz_stopped):
+    def __init__(
+        self,
+        config: CameraConfig,
+        ptz_autotracker_enabled,
+        ptz_start_time,
+        ptz_stop_time,
```

Comment:
Could we make this a single dict that holds all the syncronized values for autotrack

Reply:
These are all `mp.Value`s, so I think I'd have to use multiprocessing's `Manager` object for a dict. I agree that it would be cleaner to be a dict if `mp` wasn't involved, but a Manager implementation might be a little more messy overall. What do you think?

Reply:
I don't see why it would be any different than what is done already with the recording and audio mp values

https://github.com/blakeblackshear/frigate/blob/dev/frigate/app.py#L174-187

Reply:
Ah, gotcha. You created a new `feature_metrics` dict. I assumed you were asking about a dict inside of the existing `camera_metrics` dict.